test/harness/e2e/vm_pool.go

```diff
@@ -323,6 +323,44 @@ func SetupVMForWorker(workerID int, tempDir string, sshPortBase int) (vm.TestVMI
 	return vmPool.GetVMForWorker(workerID)
 }
 
+// StartShutoffVM checks if a specific worker VM is shutoff and starts it if needed.
+// This function is useful for ensuring VMs are running before tests begin.
+// The SSH port for each worker is calculated as: SSHPortBase (2233) + workerID
+// So worker-1 will use SSH port 2234, worker-2 will use 2235, etc.
+func StartShutoffVM(workerID int) error {
```

Comment:
I find the function name a bit confusing, I would suggest something like EnsureVMRunning or StartVMIfNeeded